### ISSUE TITLE
Apache work thread crashed while encoding TIFF...

### ISSUE THREAD

**Issue by yanhuixie** [Intervention/image#622]
**Runtime environment:**
- Centos 7, ImageMagick-6.7.8.9-15.el7_2.x86_64, imagick 3.4.3RC1, libtiff-4.0.3-14.el7.x86_64
- OSX 10.11.6, ImageMagick-6.7.5(shipped with mampstack-5.5.30-0) imagick 3.1.0RC2, libtiff.5.dylib

both two env work not properly.

**PHP Code:**

``` php
$file = $flysystem->get('/uri/to/foo.tif');
ImageManagerStatic::configure(['driver'=>'imagick']);
$img = ImageManagerStatic::make($file->read());
$img->rotate($angle, $bgColor);
$file->put($img->encode());
```

Error occurred in the encode process.

**Apache error_log:**
[Wed Sep 28 15:09:06.892256 2016] [core:notice] [pid 84979] AH00052: child pid 78541 exit signal Abort trap (6)
Assertion failed: (value>=0.0), function TIFFWriteDirectoryTagCheckedRational, file tif_dirwrite.c, line 2084.

It looks like this is the issue of libtiff, but not the intervention/image.
There are other people experiencing the same problem?

**A temporary solution:**
In short: encode a tiff file using png encoder.
Just change 

``` php
$file->put($img->encode());
```

to

``` php
$file->put($img->encode('png'));
```

This is not a good solution, but can work properly...
Looking for a better solution, any suggestion is appreciated.
